src/main/scala/io/qbeast/spark/delta/IndexStatusBuilder.scala

```diff
@@ -59,7 +58,7 @@ private[delta] class IndexStatusBuilder(
     revisionFiles
       .groupBy(TagColumns.cube)
       .agg(
-        weight(min(TagColumns.maxWeight)).as("maxWeight"),
+        min(weight(TagColumns.maxWeight)).as("maxWeight"),
```

Comment:
I read that this change is because we read a dataset which was partially written in Qbeast and Delta format?
https://github.com/osopardo1/qbeast-spark/commit/ca6c4e07c7f89c72c9956dd08373900103a5887a

Reply:
Nope, this was a bug found while trying to do so. 

In the end I decided to link that part to another issue (#121 ). Because some parameters can cause problems (`numElements` it's not easy to retrieve) or questions like what revision to use for those files.

Reply:
Understood, thanks! :)